Example/Backpack/NavigationData.swift

```diff
@@ -21,6 +21,7 @@ class NavigationData: NSObject {
     static var buttonsStoryboard = loadStoryboard(name: "Buttons")
     static var calendarStoryboard = loadStoryboard(name: "Calendar")
     static var cardStoryboard = loadStoryboard(name: "Cards")
+    static var chipStoryboard = loadStoryboard(name: "Chips")
```

Comment:
Do we still need this now?

Reply:
Kind of, but maybe we should move it. If you look at line 29 of `ChipStory` we reference this property, but maybe we should move it into `ChipStory` so it can be more self contained?

Reply:
Just tried that and worked fine, will update the PR.

Reply:
Yeah and possibly even remove the storyboard? We can just move it to Main?

Reply:
Is there a benefit to having it in a separate storyboard? I personally found it quite useful to find `Chips.storyboard` in the sidebar and go there as opposed to hunting through the main one

Reply:
At the moment we have a mixture. At some point it would be nice to put them all in one, or make them all separate, but that's out of scope for this change IMO